internal/provider/resource_aci_rest_managed_test.go

```diff
@@ -13,7 +13,7 @@ func TestAccAciRestManaged_tenant(t *testing.T) {
 	name := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:                 func() { testAccPreCheck(t) },
+		PreCheck:                 func() { testAccPreCheck(t, "apic") },
```

Comment:
do we want to run test for cloud for rest_managed?

Reply:
ok